CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 ## TO BE RELEASED
 
 
+## 1.5.1 - 6/20/2016
```

Comment:
Ooops, sorry for missing this. You need to move the release notes into the "TO BE RELEASED" section. Also the section should include notes on what/how recipes are to be run at deploy time.